schemas/openshift/managed-resource-name-1.yml

```diff
@@ -0,0 +1,23 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /openshift/managed-resource-name-1.yml
+  resource:
+    type: string
+  resourceNames:
+    type: array
+    items:
+      type: string
+    minItems: 1
+
+required:
+- resource
+- resourceNames
```

Comment:
is `resourceNames` really required?

Reply:
what would we do without it?

Reply:
only limit the kind? I think we do that most of the time. Don't we?

Reply:
this is `managedResourceNames`, not `managedResourceTypes`

Reply:
🤦